examples/lighting-app/bouffalolab/bl602/README.md

```diff
@@ -0,0 +1,83 @@
+
+## Build
+
+-   Install some additional tools(likely already present for CHIP developers):
+
+    #Linux \$ sudo apt-get install git libwebkitgtk-1.0-0 ninja-build
+
+    #Mac OS X \$ brew install ninja
+
+use GN/Ninja directly
+
+          $ cd connectedhomeip/examples/lighting-app/bouffalolab/bl602
+          $ ./go
```

Comment:
Please use some `gn gen` example instead of go.

We currently have too many custom build scripts, most go to `./scripts` but even that creates too much fragmentation. Given that this is adding a platform (12K lines) very late in the project, I am asking for more: build_examples integration.